shiny/ui/_theme_brand.py

```diff
@@ -0,0 +1,559 @@
+from __future__ import annotations
+
+import os
+import warnings
+from pathlib import Path
+from typing import Any, Optional, Union
+
+from brand_yml import Brand
```

Comment:
This could/should go under `if TYPE_CHECKING` as well

Reply:
This file isn't imported until `ui.Theme().from_brand()` is called